### PR TITLE
Tests incoming

### PR DESCRIPTION
**Pull request by MJAquarion2413**
1 working test (Create Owner test)
rest are structures to be updated as more code is completed and needs checking for edge cases or flow of structure

### CODE REVIEW

File: src/test/java/server/OwnerRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OwnerRepositoryTest {
+
+    private OwnerRepository ownerRepo;
+
+    @Test
+    void createOwner_OwnerExists() {
+        this.ownerRepo = new OwnerRepository();
```
Comment (ruiting-chen) [inline src/test/java/server/OwnerRepositoryTest.java:17]: I changed the constructor of OwnerRepository to private, so you can no longer call that method.
Instead, you can use OwnerRepository.getOwnerRepository() to get a OwnerRepository object

File: src/test/java/server/OwnerRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        ownerRepo.createOwner("Ruiting", "rtc", "abc");
+        assertFalse(ownerRepo.createOwner("Ruiting", "rtc", "abc"));
+    }
+
+    @Test
+    void createOwner_OwnerDoesNotExist() {
+        this.ownerRepo = new OwnerRepository();
```
Comment (ruiting-chen) [inline src/test/java/server/OwnerRepositoryTest.java:24]: Same issue with new OwnerRepository here

File: src/test/java/server/OwnerRepositoryTest.java (hunk 1 above)
Comment (ruiting-chen) [reply src/test/java/server/OwnerRepositoryTest.java:17]: You will need to do 
    @Before
    public void setUp() {
}
to initialize things you need for every test
and 
@After
    public void tearDown() {
} 
to remove things you don't need for the next rounds of tests
